compiler/rustc_feature/src/active.rs

```diff
@@ -417,6 +417,7 @@ declare_features! (
     /// Allows `'a: { break 'a; }`.
     (active, label_break_value, "1.28.0", Some(48594), None),
 
+    // FIXME: to be removed once rustdoc_internals is merged.
```

Comment:
(This and the below FIXME still need to be fixed.)

Reply:
We need this PR to be merged first to do so (and wait for a new nightly). :)

Reply:
Why do we need to wait?

Reply:
Because the stage 0 is compiled with an old rustc version which doesn't have that feature yet. So we need to keep the old ones around until the nightly used for stage 0 is updated.

Reply:
I don't think that's necessary. You can just do this:

```rust
#![cfg_attr(bootstrap, feature(doc_keyword)]
#![cfg_attr(not(bootstrap), feature(rustdoc_internals)]
```

(You may have to swap the `bootstrap` and `not(bootstrap)`; I always get them mixed up.)

Reply:
I tried and it failed. Also, the problem is mainly in `libcore` and `libstd` which (unless I'm wrong) needs an older rustc in any case. At least I tried with `bootstrap` and failed. But maybe I simply inverted the bootstrap?